node_exporter/node_exporter.spec

```diff
@@ -27,14 +30,19 @@ Prometheus exporter for machine metrics, written in Go with pluggable metric col
 %install
 mkdir -vp %{buildroot}%{_sharedstatedir}/prometheus
 install -D -m 755 %{name} %{buildroot}%{_bindir}/%{name}
-install -D -m 644 %{SOURCE1} %{buildroot}%{_unitdir}/%{name}.service
 install -D -m 644 %{SOURCE2} %{buildroot}%{_sysconfdir}/default/%{name}
+%if 0%{?el6} || 0%{?el5}
+mkdir -vp %{buildroot}/etc/init.d/
+install -m 644 %{SOURCE3} %{buildroot}/etc/init.d/node_exporter
+%else
+install -D -m 644 %{SOURCE1} %{buildroot}%{_unitdir}/%{name}.service
+%endif
 
 %pre
-getent group prometheus >/dev/null || groupadd -r prometheus
-getent passwd prometheus >/dev/null || \
-  useradd -r -g prometheus -d %{_sharedstatedir}/prometheus -s /sbin/nologin \
-          -c "Prometheus services" prometheus
+getent group ${group} >/dev/null || groupadd -r ${group}
```

Comment:
I think this should be `%{group}` rather than `%{group}`. Same for the user below.